dotnet/Gherkin/CucumberMessages/Pickles/PickleCompiler.cs

```diff
@@ -192,13 +192,13 @@ protected virtual StepKeywordType GetKeywordType(Step step, StepKeywordType last
     }
 
 
-    protected virtual PickleStepArgument CreatePickleArgument(Step argument)
+    protected virtual PickleStepArgument? CreatePickleArgument(Step argument)
```

Comment:
Why would the return type here be nullable? The method will definitely return an instance of PickleStepArgument.

Reply:
because on line 240 this method return null

Reply:
No, this method definitely returns a PickleStepArgument. The overload that starts on line 201 returns a possible null value, but not this one. I'm not sure how much headache is caused by such a difference.

Reply:
Ah, yes, I had overlooked that. Withdrawn.